tests/manual/monkey/src/com/oracle/tools/fx/monkey/pages/ComboBoxPage.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2022, 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package com.oracle.tools.fx.monkey.pages;
+
+import javafx.animation.KeyFrame;
+import javafx.animation.Timeline;
+import javafx.scene.control.Button;
+import javafx.scene.control.ComboBox;
+import javafx.scene.layout.VBox;
+import javafx.util.Duration;
+import javafx.util.StringConverter;
+import com.oracle.tools.fx.monkey.util.OptionPane;
+import com.oracle.tools.fx.monkey.util.TestPaneBase;
+
+/**
+ * ComboBox Page
+ */
+public class ComboBoxPage extends TestPaneBase {
+    private ComboBox control;
+
+    public ComboBoxPage() {
+        setId("ComboBoxPage");
+
+        control = new ComboBox();
+        control.getItems().setAll("0", "1", "2", "3", "4", "5", "6", "7", "8", "9");
+
+        VBox b = new VBox();
+        b.getChildren().add(control);
+
+        Button setConverterButton = new Button("Set Converter");
+        setConverterButton.setOnAction((ev) -> {
+            control.setConverter(new StringConverter() {
+                @Override
+                public String toString(Object t) {
+                    return "toString-" + t;
+                }
+
+                @Override
+                public Object fromString(String t) {
+                    return "fromString" + t;
+                }
+            });
+        });
+
+        Button changeCountButton = new Button("Change Item Count");
+        changeCountButton.setOnAction((x) -> {
+            new Timeline(
+                new KeyFrame(Duration.seconds(1.0), (ev) -> {
+                    System.out.println("2");
```

Comment:
Why do we need that debug output in this particular place? May be add some label or text field that shows the currently assumed number of items instead of relying on the system output stream? Also it would be great if there is a text somewhere which at least describes what is the purpose of the button and what is supposed to happen when you press it. If not in some designated area then at least as a tooltip on the button? I mean "Change item count" - to what? No mention of the delay so user has to either guess of why does it not change immediately or look the source code.

Reply:
good point, thanks!